aiidalab_widgets_base/databases.py

```diff
@@ -146,28 +146,30 @@ class :class:`aiidalab_widgets_base.structures.StructureManagerWidget`.
 
     structure = tl.Instance(ase.Atoms, allow_none=True)
 
-    _disable_providers = default_parameters.DISABLE_PROVIDERS
-    _skip_databases = default_parameters.SKIP_DATABASE
-    _skip_providers = default_parameters.SKIP_PROVIDERS
-    _provider_database_groupings = default_parameters.PROVIDER_DATABASE_GROUPINGS
-
     def __init__(
         self,
         embedded: bool = True,
         title: str = None,
         **kwargs,
     ) -> None:
+        from optimade_client import default_parameters, query_filter, query_provider
+
+        _disable_providers = default_parameters.DISABLE_PROVIDERS
+        _skip_databases = default_parameters.SKIP_DATABASE
+        _skip_providers = default_parameters.SKIP_PROVIDERS
+        _provider_database_groupings = default_parameters.PROVIDER_DATABASE_GROUPINGS
```

Comment:
Since those variables aren't used any further, can you directly do:

```python
disable_providers=kwargs.pop("disable_providers", default_parameters.DISABLE_PROVIDERS),
```

same for all the other cases.

Reply:
Done, thanks!